internal/rbd/nodeserver_test.go

```diff
@@ -243,7 +243,7 @@ func TestReadAffinity_GetReadAffinityMapOptions(t *testing.T) {
 	if err != nil {
 		t.Errorf("failed to marshal csi config info %v", err)
 	}
-	tmpConfPath := util.CsiConfigFile
+	tmpConfPath := t.TempDir() + "/ceph-csi.json"
 	err = os.Mkdir("/etc/ceph-csi-config", 0o600)
```

Comment:
`err = os.Mkdir("/etc/ceph-csi-config", 0o600)` remove this one as its not required any more